### PR TITLE
Add more mac platform libs

### PR DESCRIPTION
**Pull request by ilmat192**

### CODE REVIEW

File: platformLibs/src/platform/osx/builtin.def [hunk 1]
```diff
@@ -1,5 +1,6 @@
+depends = posix
```
Comment (olonho) [inline platformLibs/src/platform/osx/builtin.def]: How come?

Comment (ilmat192) [reply platformLibs/src/platform/osx/builtin.def]: Automatically :) The `set_depends` utility groups all special declarations added by Clang into each translation unit into a separate virtual header. Our current convention assumes that this virtual header is owned by the `posix` library and all other libraries (e.g. `darwin` and all new ones) import it. Thus this dependency appears.

Comment (olonho) [reply platformLibs/src/platform/osx/builtin.def]: I think dep shall be reverted and posix depend on this one, not vice versa.

Comment (ilmat192) [reply platformLibs/src/platform/osx/builtin.def]: It makes sense but I'm not sure if such dependency change can affect users or not. @SvyatoslavScherbina WDYT?

Comment (SvyatoslavScherbina) [reply platformLibs/src/platform/osx/builtin.def]: I think that this dependency is mostly technical, and it is somewhat easier to keep it in current state.

Comment (olonho) [reply platformLibs/src/platform/osx/builtin.def]: OK, let's go the current way.

**Pull request merged**